### ISSUE TITLE
untracked content: cr3gui/data/ca-bundle.crt

### ISSUE THREAD

**Issue by yparitcher** [koreader/crengine#331]
after building koreader
if i run `git status`
 i get git untracked content from cr3gui/data/ca-bundle.crt

should it be in the .gitignore?
if so i will open a PR

**Comment by NiLuJe** [comment]
Not *quite* sure where it's coming from (cURL has one, but it's not installed there), but I also noticed it the last time I did a cross build.

**Comment by Frenzie** [comment]
What exactly did you do?

**Comment by NiLuJe** [comment]
In my case, just a plain standard kobo build, and it's in there.

It's been a while, and I haven't checked the timestamps, in the insane case it'd actually be from the later cURL build and it somehow gets installed ther,e which makes no sense, but, then, CMake.

**Comment by yparitcher** [comment]
it was
`./kodev release kindlepw2`
`git status`

but i just deleted the file and rebuilt koreader and now it is not coming back.

**Comment by yparitcher** [comment]
when i delete base/build/arm-kindlepw2-linux-gnueabi/libs/libcurl.so.4
ie. rebuild curl i get the file

**Comment by yparitcher** [comment]
build/arm-kindlepw2-linux-gnueabi is a symlink to thirdparty/kpvcrlib/crengine/cr3gui/data

https://github.com/koreader/koreader-base/blob/60c8dc7f43910ccaf0457dd560ba37dceab45246/Makefile.third#L737
```
	cp $(CURL_DIR)/ca-bundle.crt $(OUTPUT_DIR)/data/ca-bundle.crt
```
is copying it to build/arm-kindlepw2-linux-gnueabi

from my build log
```
cp /koreader/base/thirdparty/curl/build/arm-kindlepw2-linux-gnueabi/curl-prefix/src/curl-build/ca-bundle.crt build/arm-kindlepw2-linux-gnueabi/data/ca-bundle.crt
```

**Comment by yparitcher** [comment]
i will leave it up to you guys the best way to aviod this :)

**Comment by Frenzie** [comment]
Probably not in this repo, imo. ;-)

**Comment by yparitcher** [comment]
is the ca-bundle.crt needed? turbo already supplies one in build/arm-kindlepw2-linux-gnueabi/common/turbo

**Comment by NiLuJe** [comment]
Yep, it's needed for HTTPS support in zsync2 (and libCURL in general).

Which we currently don't make use of right now, but, still ;).

**Comment by yparitcher** [comment]
can we use turbo's?

**Comment by NiLuJe** [comment]
No idea, we don't actually use turbo anymore, so it's probably horribly outdated (compared to libCURL's, which I know is fresh from Mozilla on each build).

**Comment by NiLuJe** [comment]
I'd probably just gitignore it here and be done with it, because dealing with setting up those paths for the various tools to find it at runtime is extremely unfun.

**Comment by NiLuJe** [comment]
Yep, turbo's copy is 7 years old :D.

**Comment by yparitcher** [comment]
> Yep, turbo's copy is 7 years old :D.

so why do we still have it?

**Comment by NiLuJe** [comment]
It's not actually used by turbo itself either, so blame its Makefile? ;).

**Comment by yparitcher** [comment]
> I'd probably just gitignore it here and be done with it, because dealing with setting up those paths for the various tools to find the bundle at runtime is extremely unfun.

You are already doing that :) you can just change the path in this line or something similar
```diff
- -----with-ca-bundle=./data/ca-bundle.crt
+ --with-ca-bundle=./common/ca-bundle.crt
```
https://github.com/koreader/koreader-base/blob/60c8dc7f43910ccaf0457dd560ba37dceab45246/thirdparty/curl/CMakeLists.txt#L30

and the copy line in Makefile.third

**Comment by NiLuJe** [comment]
After actually looking at CRe's gitignore, it turns out that doing it the other way around is consistent which what we're doing for the rest of data/ ;).

TL;DR: Killed turbo's useless copy, ignored our own here.

**Comment by Frenzie** [comment]
@NiLuJe 
> I'd probably just gitignore it here and be done with it, because dealing with setting up those paths for the various tools to find the bundle at runtime is extremely unfun.

Sure, fair enough.

Although I'm also rather of the opinion that untracked content is basically a non-issue in the first place.
